py/htm/optimization/ae.py

```diff
@@ -433,6 +433,8 @@ def __str__(self):
     def save(self):
         with open(self.lab_report + '.tmp', 'w') as file:
             file.write( str(self) )
+        if  os.path.exists(self.lab_report):
+            os.unlink(self.lab_report)
```

Comment:
does this remove the whole folder? I think so far you would append the new results to the existing report (does that make sense?)

Reply:
CC @indy-3rdman ^^

Reply:
This should only remove the `lab_report` file which is supposed to get overwritten.  My guess is that @indy added this statement because windows gives an error when you have file name conflicts?  As opposed to linux which just overwrites the old file.